content/articles/entra-identity-provider.md

```diff
@@ -0,0 +1,103 @@
+1. On the left-hand side, click on the Single-Sign On tab.
+1. Click Authorize with Microsoft Entra ID to authorize the Workspace.
+1. Fill in the Entra Tenant ID, client ID, and client secret, then click Link. You'll need to log in to the Entra Workspace.
+
+### Link the account to Entra
+
+After logging in via Entra, you will arrive back on the DNSimple Single Sign-On page with Entra SSO enabled for the account.
+If the Entra app is no longer linked (e.g. the access token is revoked), you can re-link the Entra application by clicking Reauthorize.
+
+### Access control
+
+You can set the level of DNSimple access for each member under the <label>Members and seats</label> tab in your DNSimple Account settings. For full details, visit [Domain Access Control](https://support.dnsimple.com/articles/domain-access-control/).
+
+## Linking a DNSimple user to an Entra identity
+
+When a DNSimple user exists in a DNSimple account before SSO is enabled, they will be required to link their Entra identity to verify ownership of both DNSimple and Entra user accounts. This is done on your DNSimple <label>User settings</label> page.
+To link a DNSimple user to an Entra identity:
+1. At DNSimple, use the account switcher at the top right of the page to go to your User Settings page.
+1. Scroll down to the **Identities** card, and click **Add** next to the Entra identity provider.
+1. Authenticate your Entra account.
+1. You can now log in to DNSimple using the linked Entra identity.
+
+## Logging in via Entra
+
+You'll need to [link an Entra organization to your DNSimple account](#linking-an-entra-organization-to-a-dnsimple-account) before your team members can log in via Entra SSO.
+1. To log in to DNSimple using Entra, visit https://dnsimple.com/login.
+1. Click Sign in using Entra.
+1. Enter the organization Entra domain or tenant ID and click Sign in.
+1. If you are prompted for your Entra username and password, enter them.
+1. If your credentials are valid, you will be redirected back to DNSimple and logged in.
+
+<info>
+If you are not yet a member of the DNSimple account, the account administrator will receive a notification to grant access. Once they have granted access, you will be able to see the DNSimple account's assets.
+
+If you log out of Entra, you will also be logged out of DNSimple.
+</info>
+
+## Unlinking an Entra organization from a DNSimple account
+
+1. At DNSimple, use the account switcher at the top right of the page to select the Account you want to unlink. 
+2. Once you have selected the account, go back to the account switcher and go to Account Settings. 
+3. On the left-hand side, click on the Single-Sign On tab. 
+4. Click Revoke next to the Entra SSO provider to remove the link from DNSimple to Entra.
+
+## Unlinking a DNSimple user from an Entra identity
+
+1. At DNSimple, use the account switcher at the top right of the page to go to your User Settings page.
```

Comment:
You will be met with an error if you try to do this because we don't allow it.

I get the feeling we should remove the two sections on this page relating to User Identity. It's just too confusing. Users just log in – nothing required to "link"/setup or "unlink"/remove.

It is true that you can see which identities are linked on the User Settings page, but I'm not sure how noteworthy that is.

- Linking a DNSimple user to an Entra identity
- Unlinking a DNSimple user from an Entra identity

Reply:
Google SSO is in the same boat as Okta and Entra.

This feature IS relevant for Google via social.